src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import '../../App.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    isMarried,
+    partnerName,
+    sex,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+      {isMarried
+        ? (
+          <p className="Person__partner">
+            {
+              sex === 'm'
+                ? `${partnerName} is my wife`
+                : `${partnerName} is my husband`
+            }
+          </p>
+        )
+        : <p className="Person__partner">I am not married</p>
+      }
```

Comment:
there's no need to write P tag twice, you can wrap your hole code in one P tag and then make all conditions inside, and add text by wrapping it in quotes
`
<p className="Person__partner">
        {isMarried ?  ......
  </p>
`

Reply:
agree

Reply:
Done